src/main/resources/api/openapi.yaml

```diff
@@ -281,7 +281,7 @@ components:
           type: boolean
           example: false
         globalIndex:
-          description: The global index of the box
+          description: Global index of the output in the blockchain
```

Comment:
Still not clear what this index means. Has it anything to do with block height?

Reply:
The indexer runs from block 1 to the current height.
In every block from transaction 1 to the total number of transactions in that block.
In every transaction from output 0 to the total number of outputs in that transaction.
Every output processed increases globalIndex by 1.
So the globalIndex could be thought of as a kind of 'serial number' for boxes. And also the total number of boxes ever created.

Reply:
Perfect, this text should be somewhere in the code and referred in descriptions.